### DISCUSSION TITLE
Username generator

### DISCUSSION THREAD

**Discussion by Terkyz** [padloc/padloc#643] [Ideas]
A function to generate usernames would be very useful!

**Comment by MaKleSoft** [comment]
Hi Terkyz, could you elaborate on what you mean by "generating usernames"? Are you talking about something like [this](https://github.com/padloc/padloc/issues/596)?

**Reply by Terkyz** [reply]
That's a cool feature but I wasn't referring to that, since that depends on a third-party and they are email aliases, I was referring to a function similar to the one to generate random password [but with usernames](https://i.imgur.com/8ofnHis.png), it can be useful for temporary email uses, so I don't create a username for a service that I will use for a short time that can be traceable or similar to a username of a legitimate account of mine from another service. 

Something like what LastPass [offers](https://www.lastpass.com/features/username-generator) on their web site, but built inside Padloc.
